### PR TITLE
Update multiple cookies example

### PR DESCRIPTION
**Pull request by DJDevon3**
Example for requesting cookies in a GET request. Updated to Connection Manager for 9.0

Added
- pretty serial hierarchy view
- general code cleanup and improvements
- added disable=import-error to make Pylint happy.
- imports sorted
- uses settings.toml by default
- using Connection Manager by default
- cookie emoji

Serial output example:
```
Connecting to WiFi...
✅ Wifi!
 | Fetching Cookies: https://www.adafruit.com
 | Total Cookies: 3
--------------------------------------------------------------------------------
 | 🍪 zenid=cqf625rcrijkg48l4gto; expires=Mon, 08-Apr-2024 07:22:59 GMT; Max-Age=1209600; path=/; domain=.adafruit.com; secure; HttpOnly
--------------------------------------------------------------------------------
 | 🍪 cart_count=0; expires=Mon, 08-Apr-2024 07:22:59 GMT; Max-Age=1209600; path=/; domain=.adafruit.com
--------------------------------------------------------------------------------
 | 🍪 wishlist=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; Max-Age=0; path=/; domain=.adafruit.com
--------------------------------------------------------------------------------
✂️ Disconnected from https://www.adafruit.com
Finished!
```

### CODE REVIEW

File: examples/wifi/expanded/requests_wifi_multiple_cookies.py [hunk 1]
```diff
@@ -1,36 +1,38 @@
 # SPDX-FileCopyrightText: 2022 Alec Delaney
 # SPDX-License-Identifier: MIT
-
-"""
-This example was written for the MagTag; changes may be needed
-for connecting to the internet depending on your device.
-"""
+# Coded for Circuit Python 9.0
+""" Multiple Cookies Example written for MagTag """
+# pylint: disable=import-error
```
Comment (FoamyGuy) [inline examples/wifi/expanded/requests_wifi_multiple_cookies.py:5]: I don't think this pylint disable is required is it? I checked out this branch locally and removed this line then ran `pre-commit run -a` and it succeeded without any pylint errors.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
I think we can get rid of the pylint disable comment.

File: examples/wifi/expanded/requests_wifi_multiple_cookies.py (hunk 1 above)
Comment (DJDevon3) [reply examples/wifi/expanded/requests_wifi_multiple_cookies.py]: @FoamyGuy This is good to go too. Lots of PR's to check up on.

**Review by FoamyGuy — APPROVED** [review]
Looks good. Thank you

**Pull request merged**